.prospector.yaml

```diff
@@ -1,14 +1,58 @@
 output-format: grouped
-inherits:
-  - full_pep8
-  - strictness_low
+strictness: veryhigh
+test-warnings: true
+doc-warnings: true
+max-line-length: 79
 pep8:
-  options:
-    max-line-length: 79
+  full: true
 pylint:
   enable:
     - relative-import
+  disable:
+    - C0111  # Missing %s docstring
+    - R0201  # Method could be a function
+    - W0704  # Except doesn't do anything
```

Comment:
Definitely want this one enabled. We should not have any empty except clauses. At the very least we should usually be logging them.